routes/index.js

```diff
@@ -32,4 +31,13 @@ router.post('/plaid/transaction', plaid.handlePlaidTransaction);
 router.post('/budget/', budget.createCategory);
 router.post('/budget/edit', budget.editCategory);
 router.post('/budget/delete', budget.deleteCategory);
+
+const transactions = require('./transactions');
+router.post('/transactions/addTransaction', transactions.addTransaction);
```

Comment:
You don't really need to do `transactions/addTransaction`, you could do `transactions/add`. The second transaction is pretty repetitive.

Reply:
Same for all the other routes